src/commands/save-audit-logs.mts

```diff
@@ -0,0 +1,83 @@
+// Command to save audit logs from the past week
+// Logs are saved to a file locally which is also posted to the channel
+// Will run weekly, but can still be invoked manually by the user
+
+import { CommandInteraction, Guild, SlashCommandBuilder, TextChannel } from "discord.js";
+import { Command } from "../types.mjs";
+import fs from "fs";
+import path from "path";
+
+export default {
+    data: new SlashCommandBuilder().setName("save-audit-logs").setDescription("Save audit logs from the past week."),
```

Comment:
Could you add a date parameter on when save audit logs since?

Reply:
Taken care of in another commit